website/static/worker.js

```diff
@@ -5,20 +5,30 @@
 self.global = self;
 self.util = {};
 self.path = {};
+self.path.resolve = function() {
+  return "";
+};
 self.Buffer = {
   isBuffer: function() {
     return false;
   }
 };
 self.constants = {};
 // eslint-disable-next-line
-module$1 = module = path = os = crypto = {};
+module$1 = module = os = crypto = {};
 self.fs = { readFile: function() {} };
 // eslint-disable-next-line no-undef
 os.homedir = function() {
   return "/home/prettier";
 };
-self.process = { argv: [], env: { PRETTIER_DEBUG: true }, version: "v8.5.0" };
+self.process = {
```

Comment:
This stuff is getting a bit out of hand. I wonder if using browserify or rollup-plugin-node-builtins might be better?

Reply:
babelify... 😰

![image](https://user-images.githubusercontent.com/8341033/34637457-4616242e-f2f2-11e7-9b67-4fdeda8c2930.png)

Reply:
We should find a way to remove those dependencies for the browser bundle actually